backend/RoomPlanner/room-planner/pom.xml

```diff
@@ -43,6 +46,30 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-core</artifactId>
+			<scope>test</scope>
+        </dependency>
+		<dependency>
```

Comment:
I am seeing this two times here
		<dependency>
			<groupId>org.springframework.security</groupId>
			<artifactId>spring-security-core</artifactId>
			<version>RELEASE</version>
			<scope>compile</scope>
		</dependency>
Is this right?